### PR TITLE
Improve docker

### PR DESCRIPTION
**Pull request by mykiwi**
I was playing with your repo, so I made some updates:

- Optimize Docker image: 42.6MB into 20.3MB
- Optimize PHP autoloader
- Update PHP dependencies

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+	php7-cli \
+	php7-iconv \
+	php7-mbstring
+
+COPY --from=0 /app/ /usr/local/src/docker-hostmanager
 
 ENV HOSTS_FILE=/hosts
 
-ENTRYPOINT ["/usr/local/bin/docker-hostmanager"]
+CMD ["/usr/local/src/docker-hostmanager/bin/docker-hostmanager"]
```
Comment (iamluc) [inline Dockerfile]: Not sure about changing the entrypoint... It will beak for users that have parameters/arguments

**Review by iamluc — CHANGES_REQUESTED** [review]
Nice! Thank you.
Other than the entrypoint, LGTM :+1: 

But could you target the `develop` branch ? It will allows us to test it safely

**Comment by mykiwi** [comment]
Ok :smiley:

**Comment by mykiwi** [comment]
Done :)

**Comment by iamluc** [comment]
Thank you!

**Pull request merged**